db_scripts/mosip_esignet/deploy.sh

```
@@ -1,65 +1,54 @@
-	exit 0
+  echo "Database '$MOSIP_DB_NAME' already exists, please reinitialise after deleting or renaming the same."
+  exit 0
 else
-	echo "eSignet database was not found. Running the db scripts to create and initialize the eSignet database."
+  echo "eSignet database was not found. Running the db scripts to create and initialize the eSignet database."
 fi
 
 ## Terminate existing connections
 echo "Terminating active connections"
-CONN=$(PGPASSWORD="$SU_USER_PWD" psql -v ON_ERROR_STOP=1 --username="$SU_USER" --host="$DB_SERVERIP" --port="$DB_PORT" --dbname="$DEFAULT_DB_NAME" -t -c "SELECT count(pg_terminate_backend(pg_stat_activity.pid)) FROM pg_stat_activity WHERE datname = '$MOSIP_DB_NAME' AND pid <> pg_backend_pid()"; exit;)
+CONN=$(PGPASSWORD=$SU_USER_PWD psql -v ON_ERROR_STOP=1 --username=$SU_USER --host=$DB_SERVERIP --port=$DB_PORT --dbname=$DEFAULT_DB_NAME -t -c "SELECT count(pg_terminate_backend(pg_stat_activity.pid)) FROM pg_stat_activity WHERE datname = '$MOSIP_DB_NAME' AND pid <> pg_backend_pid()";exit;)
```

Shellcheck (0.11.0)

[warning] CONN appears unused. Verify use (or export if used externally).

(SC2034)